roles/apim-km/templates/carbon-home/repository/conf/user-mgt.xml.j2

```diff
@@ -27,7 +27,7 @@
             <!-- By default users in this role sees the registry root -->
             <Property name="isCascadeDeleteEnabled">true</Property>
             <Property name="initializeNewClaimManager">true</Property>
-            <Property name="dataSource">jdbc/WSO2CarbonDB</Property>
+            <Property name="dataSource">WSO2_UM_DB</Property>
```

Comment:
This should be `jdbc/WSO2UM_DB`.

Reply:
Fixed https://github.com/wso2/ansible-apim/pull/49/commits/6464f58613fc352dd7b9fcc50d0bb1c1f9d8a6e1